plugins/org.eclipse.n4js/xsemantics-gen/org/eclipse/n4js/xsemantics/InternalTypeSystem.java

```diff
@@ -4,12 +4,10 @@
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- *
+ * 
```

Comment:
would it be possible to avoid the trailing space? (for consistency with copyright headers in other files)